src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -417,6 +418,24 @@ def test_update_run(self, pf, randstr: Callable[[str], str]):
         assert run.description == new_description
         assert run.tags["test_tag"] == test_mark
 
+    def test_cancel_run(self, pf, runtime: str, randstr: Callable[[str], str]):
```

Comment:
no recording? is there any error you met with this operation? maybe I can help take a look?

Reply:
Simply forgot the add lol. Seems no error happened. Updated the recording.